lms/djangoapps/course_wiki/middleware.py

```diff
@@ -29,7 +29,7 @@ def _redirect_from_referrer(self, request, wiki_path):
         """
         redirect to course wiki url if the referrer is from a course page
         """
-        course_id = course_id_from_url(request.META.get('HTTP_REFERER'))
+        course_id = course_id_from_url(request.headers.get('referer'))
```

Comment:
Introduced in `Django 2.2` [[Release notes](https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.headers)]

Reply:
This change was causing tests to fail in `lms-2` shard so reverted the change.